guides/source/configuring.md

```diff
@@ -1733,177 +2080,36 @@ The default is `:rails_storage_redirect`.
 
 Can be used to alter the way ffmpeg generates video preview images.
 
-With `config.load_defaults 7.0`, the default is:
+The default value depends on the `config.load_defaults` target version:
 
-```ruby
-config.active_storage.video_preview_arguments = "-vf 'select=eq(n\\,0)+eq(key\\,1)+gt(scene\\,0.015),loop=loop=-1:size=2,trim=start_frame=1' -frames:v 1 -f image2"
-```
-
-Which has the following behavior:
-
-1. `select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015)`: Select the first video frame, plus keyframes, plus frames that meet the scene change threshold.
-2. `loop=loop=-1:size=2,trim=start_frame=1`: Use the first video frame as a fallback when no other frames meet the criteria by looping the first (one or) two selected frames, then dropping the first looped frame.
+| Starting with version | The default value is |
+| --------------------- | -------------------- |
+| (original)            | `"-y -vframes 1 -f image2"` |
+| 7.0                   | `"-vf 'select=eq(n\\,0)+eq(key\\,1)+gt(scene\\,0.015)"`<sup><mark><strong><em>1</em></strong></mark></sup> <br> `+ ",loop=loop=-1:size=2,trim=start_frame=1'"`<sup><mark><strong><em>2</em></strong></mark></sup><br> `+ " -frames:v 1 -f image2"` <br><br> <ol><li>Select the first video frame, plus keyframes, plus frames that meet the scene change threshold.</li> <li>Use the first video frame as a fallback when no other frames meet the criteria by looping the first (one or) two selected frames, then dropping the first looped frame.</li></ol> |
 
 #### `config.active_storage.multiple_file_field_include_hidden`
 
 In Rails 7.1 and beyond, Active Storage `has_many_attached` relationships will
-default to _replacing_ the current collection instead of _appending_ to it. To
-support submitting an _empty_ collection, render an auxiliary hidden field
-similar to how Action View Form Builder render checkbox elements.
+default to _replacing_ the current collection instead of _appending_ to it. Thus
+to support submitting an _empty_ collection, when `multiple_file_field_include_hidden`
+is `true`, the [`file_field`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field)
+helper will render an auxiliary hidden field, similar to the auxiliary field
+rendered by the [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box)
+helper.
+
+The default value depends on the `config.load_defaults` target version:
+
+| Starting with version | The default value is |
+| --------------------- | -------------------- |
+| (original)            | `false`              |
+| 7.0                   | `true`               |
 
 ### Configuring Action Text
 
 #### `config.action_text.attachment_tag_name`
 
 Accepts a string for the HTML tag used to wrap attachments. Defaults to `"action-text-attachment"`.
 
-### Results of `config.load_defaults`
```

Comment:
Actually, I don't think we should remove this list. Now, to know which defaults are applied on 7.0, I need to read the entire page, not just this section. I think we should keep it.

Reply:
Ok, I can add it back.  However, the reason I removed it was so that contributors would not have to remember to update the default values in two places, since that would be more error-prone.

Reply:
I agree it is harder for contributors, but that is on us to better review the PRs. In terms of usability for our user the tables are great, but knowing which configs are applied when `load_default "7.1"` is called, in a single place also is. We can have both.

Reply:
Another thought: it seems like seeing the whole list at a glance might be more useful in the Upgrading guide rather than the Configuration guide.

To ease maintenance, what about just linking the appropriate new framework defaults file (e.g. [`new_framework_defaults_7_0.rb.tt`](https://github.com/rails/rails/blob/v7.0.0/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt)) from the Upgrading guide?

Reply:
I don't think we add everything in the new frameworks default file, so I don't think that would work. Not sure if the best place is the upgrading guide, maybe it is, but them people will have to change two files instead of 1, and if we are concerned about people not documenting it, isn't easier to forget to update two files instead of 1?

Reply:
My instinct is that if a new default isn't mentioned in the new framework defaults file, then it probably deserves its own mention in the Upgrading guide.  But maybe there is a counter example that I am failing to think of?

If I do add the full list back, would you prefer it where it was, or in [the new section](https://github.com/rails/rails/pull/44038/files#diff-210d0f245e618b7f8c6fcb53f518bd7fd343ce7e5d8c609ecfdb149418beca19R53) near the top?

Reply:
In that section near the top would be better.